### PR TITLE
Lists sets by most recently created

### PR DESCRIPTION
**Pull request by suzkiee**
# Description
Completes User Story 6.

## Fixes # (issue)
 This branch adds logic to the Crystal Set Index that lists the sets by most recently created. 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Green tests written based off the user stories

### CODE REVIEW

File: spec/features/crystal_sets/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    expect(page).to have_content(set.collection_name)
+    expect(page).to have_content(set.limited_edition)
+    expect(page).to have_content(set.created_at.strftime('%m/%d/%y'))
+  end
+
+  it 'shows crystal sets in order of date created' do
+    visit '/crystal_sets'
+
+    expect("Intuitively Chosen Raw Crystals Collection").to appear_before("Raw Crystal Chunks - 28pc Collection", only_text: true)
```
Comment (LawrenceWhalen) [inline spec/features/crystal_sets/index_spec.rb:27]: I would check this assertion by printing the whole time stamp. I found that spec was running so fast that all of the sets had the same creation time. You can add a `created_at` to the create! command to be sure that they are appearing in the correct order. The `save and open page` command helped me a lot here.

File: spec/models/crystal_set_spec.rb [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
- it {should have_many :crystals}
+  it {should have_many :crystals}
+
+  describe 'instance methods' do
+    set_1 = CrystalSet.create!(collection_name: "Raw Crystal Chunks - 28pc", limited_edition: false, inventory: 1000)
+    set_2 = CrystalSet.create!(collection_name: "Intuitively Chosen Raw Crystals", limited_edition: true, inventory: 250)
+    sets = [set_1, set_2]
+    
+    expect(sets.order).to eq([set_2, set_1])
+  end
```
Comment (LawrenceWhalen) [inline spec/models/crystal_set_spec.rb:17]: If I am reading the rspec documentation correctly, you've already tested this in your feature spec tests. You should only need model testing for methods not in the controller class.

**Review by LawrenceWhalen — APPROVED** [review]
Commits look great, only had a couple of questions about testing protocols.

File: spec/features/crystal_sets/index_spec.rb (hunk 1 above)
Comment (suzkiee) [reply spec/features/crystal_sets/index_spec.rb:27]: Ahh that's a really good point! I'm also a little confused on the `save_and_open_page` command. I thought it was something akin to using pry in your test just to make sure that your test runs properly. Do you need to keep it in?

File: spec/models/crystal_set_spec.rb (hunk 2 above)
Comment (suzkiee) [reply spec/models/crystal_set_spec.rb:17]: Ahhhh thank you for pointing that out!

**Pull request merged**